manifests/add-cluster/add-cluster.sh

```diff
@@ -37,10 +37,16 @@ STATUS_CODE=$(curl -X POST ${INGRESS_URL}/app-proxy/api/clusters \
   -H 'Authorization: '${CSDP_TOKEN}'' \
   -d '{ "name": "'${CONTEXT_NAME}'", "kubeConfig": "'${KUBE_CONFIG_B64}'" }' \
   -s -o response -w "%{http_code}")
-cat response
 echo "STATUS_CODE: ${STATUS_CODE}"
+cat response
+echo
+
+if [[ $STATUS_CODE == 000 ]]; then
+  echo "error sending request to runtime"
+  exit 1
+fi
 
-if [ $STATUS_CODE -ge 300 ]; then
+if [[ $STATUS_CODE -ge 300 ]]; then
```

Comment:
I think generally you should check for `-ge 400` because 300-399 contains a bunch of valid responses no?